### ISSUE TITLE
Abholen ohne Karte?

### ISSUE THREAD

**Issue by mbirth** [probonopd/packstation-barcode#2]
Da das hier irgendwie die einzige Anlaufstelle für sowas zu sein scheint:

Hat sich hier mal jemand mit der Berechnung der Abholcodes beschäftigt? Neuerdings kann der Postbote ein Benachrichtigungskärtchen in den Briefkasten werfen. Darauf ist ein Strichcode (sowie die Packstation und Empfängername) gedruckt. Mit diesem Strichcode, dem Empfängernamen und einer "Unterschrift", die man per Finger auf dem Touchscreen malen soll, kann man sein Paket aus der Packstation entnehmen.

Nun ist mir aufgefallen, dass der Trackingcode: 

```
JVGL05952619xxxxxxxxxxx
```

die Abholnummer

```
1593xxxxxxxxxxx6
```

ergeben hat. (Die `x` waren identisch.) Das sieht mir nach einem Leitcode für die Packstation und einer Prüfziffer aus.

Da die Benachrichtigungskarte leider erst am übernächsten Werktag auf dem Postweg kommt, man im Tracking aber schon die Packstation sehen kann, wäre es doch prima, wenn jemand einen Weg finden würde, diesen Abholcode zu berechnen.

**Comment by mbirth** [comment]
Hier scheinbar das gleiche Spiel:

![](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c7/Benachrichtigungskarte_f%C3%BCr_DHL-Paket%2C_Filiale.jpg/640px-Benachrichtigungskarte_f%C3%BCr_DHL-Paket%2C_Filiale.jpg)

```
Sendungsnummer: 396013134140
Abholnummer: 1691960131341405
```

Und das, obwohl es dort in einer Filiale abgeholt werden soll.

[Und hier noch ein Beispiel.](http://philaseiten.wavecdn.net/up/3338231/6/8/99e3be30_h.jpeg)

```
Sendungsnummer: JJD1405085588618
Abholnummer:     0753050855886187
```

**Comment by probonopd** [comment]
> Da die Benachrichtigungskarte leider erst am übernächsten Werktag auf dem Postweg kommt

Ist das ein Aprilscherz?

**Comment by mbirth** [comment]
> Ist das ein Aprilscherz?

Ich wünschte, es wäre so. Aber wenn der Fahrer keine Lust mehr hat, stopft er halt alles in die Packstation, druckt die Kärtchen aus und gibt die im Depot auf den Postweg. So hatten wir letzten Samstag im Tracking gesehen, dass die Sendung in der Packstation gelandet ist, hatten aber erst gestern Abend die Karte im Postkasten. (Ja, Montag war Pfingsten, daher war das diesmal "schnell".) Und manchmal verzögert sich das halt noch um einen weiteren Tag.

**Comment by probonopd** [comment]
Warum kommt eigentlich in so einem Abholkarte-Fall keine SMS mit einer TAN (wie bei einer "normalen" Packstation-Sendung)? Vermutlich, weil die PostNummer nicht auf der Sendung stand?

**Comment by mbirth** [comment]
Genau! Die Sendung ist keinem Packstation-Kunden zugeordnet, daher läuft das ohne mTAN, SMS, o.ä..
